source/runtime.cpp

```diff
@@ -960,7 +968,11 @@ namespace reshade
 	}
 	void runtime::save_preset(const filesystem::path &path) const
 	{
-		ini_file preset(path);
+		save_preset(path, path);
+	}
+	void runtime::save_preset(const filesystem::path &path, const filesystem::path &save_path) const
+	{
+		ini_file preset(path, save_path);
```

Comment:
See above. There should not be a need for the original preset path, since all required information are set in this function anyway.

Reply:
If there is no overload, Completely disabled effect settings will be lost.